### DISCUSSION TITLE
Bed Articulation vs. Toolhead articulation

### DISCUSSION THREAD

**Discussion by JasperCohen** [FreddieHong19/Open5x#23] [General]
Hi Freddie,

I'm curious what led you to this approach. Thanks!

**Comment by FreddieHong19** [comment]
I kept the nozzle fixed vertically, so i) the material is always extruded downward, ii) the rotary bed does not need to unwind and therefore the tool path can be optimised and most importantly, iii) since the bed is intendent to the machine hardware, we can implement the design to fit various other existing 3d printers

**Reply by harasmeant** [reply]
Hey, I'm modifiying your aproach on the bed articulation. Would only need some help with the code to be able to modifyit in order to be able to use it. Pls contact me.
